### ISSUE TITLE
Admin settings for each grant:

### ISSUE THREAD

**Issue by YorgunKelebek** [goofballLogic/g4a-spa#8]
- deadline for applications, after which grant closes for submissions
- total funds available, when reached (cumulative value for awarded applications) grant closes for submissions
- other settings?

**Comment by goofballLogic** [comment]
"when reached (cumulative value for awarded applications) grant closes for submissions" - this could be useful but perhaps it would be more useful to prevent further applications being approved for funding until:

```
[amount being approved] <= [total initial funds] - [sum of accepted offers] - [sum of outstanding offers]
```
?

**Comment by goofballLogic** [comment]
I think perhaps we should separate out things to do with "tenancy" - e.g. (1) ability to name your tenancy something other than "Grants by Hakan" (2) ability to invite other administrators to your tenancy

**Comment by YorgunKelebek** [comment]
Yes, I agree on both accounts - these will be a must for anyone wanting to use the system.

**Comment by goofballLogic** [comment]
Split out the two remaining features so placing back in Test

**Comment by YorgunKelebek** [comment]
I personally would like to see that the deadline could be set for a specific time of the day. This is a common practice with similar application type processes.

**Comment by goofballLogic** [comment]
> I personally would like to see that the deadline could be set for a specific time of the day. This is a common practice with similar application type processes.

The issue which always came up with Flexi-Grant was which timezone this deadline applied to. Do you have an opinion on that?

**Comment by goofballLogic** [comment]
And in fact I believe Flexi-Grant doesn't really prevent submission of applications after the exact time of day although they are flagged up in some way if I recall correctly.

**Comment by YorgunKelebek** [comment]
Time zone for application deadlines would be a tenancy specific setting in my opinion.

**Comment by goofballLogic** [comment]
Or we could allow specification of the time zone when setting the date - defaulting to the manager's locale (or in the future a tenancy setting).

**Comment by YorgunKelebek** [comment]
> Or we could allow specification of the time zone when setting the date - defaulting to the manager's locale (or in the future a tenancy setting).

I personally don't think it should vary within the tenancy - setting it to the managers locale could create variations and therefore complications/misunderstandings (remote working in different time zones for example)...
